homeassistant/components/plugwise/climate.py

```diff
@@ -62,6 +63,10 @@ def __init__(
         self.gateway_data = coordinator.data.devices[gateway_id]
         # Determine supported features
         self._attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
+        if HVACMode.OFF in self.hvac_modes:
+            self._attr_supported_features |= ClimateEntityFeature.TURN_OFF
+            if any(mode for mode in self.hvac_modes if mode != HVACMode.OFF):
+                self._attr_supported_features |= ClimateEntityFeature.TURN_ON
```

Comment:
Sorry @frenck that is not correct. The HVACMode.OFF is only present for an Adam, not for an Anna.

Reply:
I think you read my comment wrong... 

When we get to the part this comment is on, it was already determined `OFF` is supported (line 66). The lines I comment on (68-69) are about the next step: Checking if `ON` is supported, which will always be the case because of:

https://github.com/home-assistant/core/blob/39c41a21f9f0e70ed593f7711d46d7aa9fbac290/homeassistant/components/plugwise/climate.py#L157-L158

Reply:
Ok, understood, thanks.